docs/xbcloud-binary-fifo-datasink.md

```diff
@@ -0,0 +1,70 @@
+# FIFO data sink
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements a new data sink that uses the first in, first out (FIFO) method. With the `FIFO` data sink feature, users with a streaming capacity of 10Gbps (typically on a Local Area Network (LAN)) can benefit from faster backups by streaming data in parallel to object storage.
+
+## FIFO data sink options
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements the following options:
+
+* `--fifo-streams=#` - specifies the number of FIFO files to use for parallel data stream. To disable FIFO data sink and send stream to STDOUT, set `--fifo-streams=1`. The default value is `1` (STDOUT) to ensure the backward compatibility. The `--fifo-streams` value must match on both the XtraBackup and xbcloud sides.
+* `--fifo-dir=name` - specifies a directory to write Named Pipe.
+* `--fifo-timeout=#` - specifies the number of seconds to wait for the other end to open the stream for reading. The default value is `60` seconds.
+
+## How to run FIFO data sink
+
+To run FIFO data sink, you need to either invoke the following commands in separate terminal sessions: 
+
+```{.bash data-prompt="$"}
+$ xtrabackup --backup --stream --fifo-streams=2 --fifo-dir=/tmp/fifo
+```
+
+and
+
+```{.bash data-prompt="$"}
+$ xbcloud put --fifo-streams=2 --fifo-dir=/tmp/fifo full
+```
+
+or run xtrabackup in the background.
+
+## Stream to object storage
+
+When taking a backup, you can save the files locally or stream the files to either different servers or object storage. 
+
+Before [Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md), XtraBackup streams data to object storage writing to STDOUT (a pipe) and using [`xbcloud`](xbcloud-binary-overview.md) to read from STDIN (standard input device). 
+
+This is how it works:
+
+XtraBackup spawns multiple copy threads and each copy thread reads a data chunk from a specific file. Then each copy thread writes the data chunks to a pipe (STDOUT). An xbcloud read thread reads each chunk of STDIN data. Then xbcloud uploads the chunks to an object storage using an async request, and adds a callback to an event handler list. The xbcloud event handler executes the callback depending on the response from the object storage (success or failure).
+
+![image](_static/backup-streamed-to-object-storage.png)
+
+XtraBackup streaming using STDOUT has streaming capacity of 1.8Gbps. This capacity is sufficient for streaming data using Wide Area Network (WAN) into, for example, Amazon Web Services or Google Cloud Platform. If you stream backups to Amazon S3 compatible storage using LAN with streaming capacity of 10Gbps, XtraBackup can use multiple FIFO streams to stream the backups faster. 
```

Comment:
The streaming capacity is 1.8G for XtraBackup using STDOUT. 
...
I'm not sure the following sentence belongs in the earlier version explanation:
If you stream backups to Amazon S3 compatible storage using LAN with streaming capacity of 10Gbps, XtraBackup can use multiple FIFO streams to stream the backups faster.